test/compose/test-compose

```diff
@@ -229,7 +231,7 @@ function start_service() {
         sleep 1
         _timeout=$(( $_timeout - 1 ))
     done
-    cat $WORKDIR/server.log
+    # cat $WORKDIR/server.log
```

Comment:
Why remove this?

Reply:
This will clutter the log unreadable because it is always printed for each test run.

Reply:
Check again. It only triggers if podman server doesn't come up.

Reply:
You are correct, I missed the early return there.